commit-stream.js

```diff
@@ -1,4 +1,5 @@
-const through2 = require('through2')
+const through2 = require('through2'),
+stripAnsi = require('strip-ansi');
```

Comment:
comma first, no semicolon, indent `stripAnsi` to line up at the start of `through2`, align `=`

```js
const through2  = require('through2'),
    , stripAnsi = require('strip-ansi')
```